atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -563,6 +564,13 @@ def get_build(self, metadata):
                 else:
                     extra['filesystem_koji_task_id'] = task_id
 
+        media_types = []
```

Comment:
This should also append `['application/json']` if pulp_push was running

Reply:
I think pulp_pull already does that when reporting media types. For prod builds, pulp_push runs in the same OpenShift Build as pulp_pull.
https://github.com/projectatomic/atomic-reactor/blob/1.6.26.3/atomic_reactor/plugins/post_pulp_pull.py#L105-L109

Reply:
Oh, I see, I assumed `koji_promote` code is more recent. Its not needed then, yes